src/scores/spatial/fss_impl.py

```diff
@@ -302,8 +307,8 @@ def fss_2d_single_field(
 
 
     The caller is responsible for making sure the input fields are in the 2-D
-    spatial domain. (Although it should work for any `np.array` as long as it's
-    2D, and `window_size` is appropriately sized.)
+    spatial domain. (Although it should work for any ``np.array`` as long as it's
```

Comment:
It looks like there are double-backticks here, is that intended?

Reply:
yup, otherwise it isn't rendered as monospace